accepted/0000-caasp-integration.md

```diff
@@ -0,0 +1,298 @@
+
+  For this reason, the CaaS Platform [supports deployment on](https://susedoc.github.io/doc-caasp/beta/caasp-deployment/single-html/#_platform):
+
+  - VMware ESXi
+  - SUSE OpenStack Cloud 8
+  - [Bare-metal deployment](https://susedoc.github.io/doc-caasp/beta/caasp-deployment/single-html/#_deployment_on_bare_metal)
+
+  CaaS Platform also provides templates for the above providers that can be found in the `skuba` package (in `/usr/share/caasp/{AutoYaST, terraform}`). For VMware and SUSE OpenStack Cloud, the template is provided via Terraform. The up-to-date version of these templates can be found at:
+  - for VMware: https://github.com/SUSE/skuba/tree/master/ci/infra/vmware
+  - for SUSE OpenStack Cloud 8: https://github.com/SUSE/skuba/tree/master/ci/infra/openstack
+  - for AutoYaST: https://github.com/SUSE/skuba/blob/master/ci/infra/bare-metal/AutoYaST.xml
+
+  One additional scenario would be to use [an existing node running SUSE Linux Enterprise 15](https://susedoc.github.io/doc-caasp/beta/caasp-deployment/single-html/#_deployment_on_existing_sles_installation). In this particular case, we should check that the CaaS Platform requirements are satisfied before offering this node to be part of the cluster.
+  These checks can be done using Salt grains. 
+
+  ##### Templating selection
+  
+  SUSE Manager has an existing integration with VMWare (Virtual Host Manager). This integration is read-only, as SUSE Manager can show the guests running in the VMWare host and display information about the guests.
+  Currently, there is no any integration with SUSE OpenStack Cloud 8.
+  Additionally, SUSE Manager has not currently any integration with `terraform`.
+
+  SUSE Manager supports autoinstallation via AutoYaST.
+
+  For the reasons above, it looks promising to invest in two directions for fulfilling the integration in the deployment phase:
+  1. Existing node: SUSE Manager must filter the eligible nodes that satisfy all CaaS Platform node requirements among the already registered Salt minions already available
+  2. AutoYaST deployment: in case the user wants to deploy additional autoinstalled node, SUSE Manager should guide the user into setting up as many nodes of the cluster by autoinstalling them using the AutoYaST profile provided by `skuba`.
+
+  **NOTE**: skuba also provides templates for AWS and libvirt, but those are not supported at the moment of writing.
+
+  ##### The Management Node
+
+  CaaS Platform requires a "Management Node" to deploy itself. The Management node must run SUSE Linux Enterprise 15 SP1 or an OpenSUSE equivalent (TODO: check with CaaSP team: there aren't repos for OpenSUSE).
```

Comment:
The pattern used to install CaaSP is in the SLE repos at the moment, mentioned later in the RFC.

Reply:
OK, removing openSUSE.

Reply:
In the future there will be a CaaSP Community that will work on openSUSE. Maybe we should include in the RFC that these will the initial pairs supported?
CaaSP 4.0 + SUSE Manager 4.0
CaaSP Community 4.0 + Uyuni Server 4.0

But not CaaSP 4.0 on Uyuni Server 4.0 (not that I expect anyone to try that)

Reply:
Added to the RFC